basic/codeship-pro/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # https://documentation.codeship.com/pro/languages-frameworks/nodejs/
 
 # use Cypress provided image with all dependencies included
-FROM cypress/base:10
+FROM cypress/base:latest
```

Comment:
Cypress v12 dropped node 12 support. This is an example for users. Updated to pull latest.